Automattic/sensei
# Round course progress to integer

Removes decimal points in Course progress bar headings

### Changes proposed in this Pull Request

* Rounded course progress headings to show the nearest integer value

### Testing instructions

- Create a course with multiple lessons
- Enable learning mode
- Create a page with a course list block in it, selecting a pattern that has course progress bar
- Have a progress bar on the course you've created too
- Complete a random number of lessons of that course with a user and make sure the percentage of completion is not an integer (like 1 of 3) in real life
- Now go the progress bars in all those places and make sure the progress is shown as a number rounded to the nearest integer

### Screenshot / Video
![image](https://user-images.githubusercontent.com/6820724/206016992-32196b6b-7532-4e17-b630-120f5b84e530.png)
